### ISSUE TITLE
Brand new Uno project crashes in Android in Release mode before displaying any pages

### ISSUE THREAD

**Issue by FrancoisM** [nventive/Uno#853]
## I'm submitting a...
Bug report (I searched for similar issues and did not find one) 

## Current behavior
In release, the app crashes on Android device after the blank screen before opening the main page.

## Expected behavior
The main page should display as in Debug without any crashed.

## Minimal reproduction of the problem with instructions
Brand new project from Uno template.

## Environment

Nuget Package: Uno.UI

Package Version(s): 1.44.0

Affected platform(s):
- [ ] iOS
- [x] Android
- [ ] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [ ] 2017 (version: )
- [x] 2019 (version:  16.0.3)
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )

**Comment by jeromelaban** [comment]
Thanks for reporting the issue! I tried creating the issue with the same repro as yours, and I cannot reproduce the issue.

Could you provide the exact solution you've created, as we may have missed something there. Also which version of Android are you running on ?

Thanks!

**Comment by FrancoisM** [comment]
Thank you for looking into it.

Project that crashes:
https://github.com/FrancoisM/ReproUnoTemplateCrashInReleaseInAndroid

The device is a Samsung S9 running Android version: 9

**Comment by jeromelaban** [comment]
Thanks, could you also paste the exception you're getting from AppCenter ? It is the same that the one on gitter ?

**Comment by carldebilly** [comment]
I successfully reproduced the problem on a Google Moto G7 device.

The problem is related to the "LLVM AOT Optimization". You may disable it to fix your problem.

Microsoft knows about this problem and it has already been fixed in latest _Preview_ version of Visual Studio 2019 (more details [here](https://github.com/xamarin/xamarin-android/issues/2960#issuecomment-486028519)).

If you need to use it from the current version of VisualStudio, please set this in your _csproj_  file: `<EnableLLVM>false</EnableLLVM>`

**Comment by FrancoisM** [comment]
Hi @carldebilly, how did you find out? Once you reproduced the issue, where did you get any info about the problem? Which log did you look into? What error did you see and were?
Thank you for your help, I'd like to learn from this 👍

**Comment by carldebilly** [comment]
Hello @FrancoisM, the problem was obviously coming from something done only in _Release_ mode. Usually it's related to eiter _linking_ or the _AOT_ processing.

I deactivated the _AOT_ and the problem were gone... so my last check was to check if it's related to the LLVM optimisation and bingo!  After that a quick search on the web and I found the link I sent to you.

Last step was to check in 2019 Preview and I confirmed the problem was solved there... :-)
